source/qml/Library/RendererQml/AdaptiveCardQmlRenderer.cpp

```diff
@@ -620,21 +621,58 @@ namespace RendererQml
 		{
 			contentItemTag->Property("placeholderText", "\"" + input->GetPlaceholder() + "\"");
 		}
-		contentItemTag->Property("background", backgroundTag->ToString());
+
+		auto textBackgroundTag = std::make_shared<QmlTag>("Rectangle");
+		textBackgroundTag->Property("color", "'transparent'");
+		contentItemTag->Property("background", textBackgroundTag->ToString());
+		contentItemTag->Property("onEditingFinished", Formatter() << "{ if(Number(text) < " << inputId << ".from || Number(text) > " << inputId << ".to){\nremove(0,length)\nif(" << inputId << ".hasDefaultValue)\ninsert(0, " << inputId << ".defaultValue)\nelse\ninsert(0, " << inputId << ".from)\n}\n}");
```

Comment:
What is `from` is not provided? Does it consider as `0`?

Reply:
yes in this case
because of input->SetMin(0) 
if we replace 0 with INT_MIN then number.from will be -2147483648